ldsc.py

```diff
@@ -599,6 +599,7 @@ def ldscore(args, log):
         log.log(header)
         log.log('Beginning analysis at {T}'.format(T=time.ctime()))
         start_time = time.time()
+        print(args.ref_ld)
```

Comment:
We should prefer `log`, right?

This is maybe the only change I think we really need to make.